tests/unit/workflows/test_workflows_tasks_actions.py

```diff
@@ -322,6 +323,19 @@ def test_halt_record_accepts_custom_msg():
     assert eng.msg == 'bar'
 
 
+def test_preserve_root():
+    config = {
+        'FEATURE_FLAG_ENABLE_MERGER': True
+    }
+
+    with patch.dict(current_app.config, config):
+        obj = MockObj({'foo': 'bar'}, {})
+        eng = MockEng()
+
+        assert preserve_root(obj, eng) is None
```

Comment:
hmm why we need this assertion? the following one should be enough :)

Reply:
I followed the style of the test above. I can remove it tho :)